providers/standard/docs/changelog.rst

```diff
@@ -35,6 +35,43 @@
 Changelog
 ---------
 
+1.5.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add Human-in-the-loop logic to core Airflow and implement 'HITLOperator', 'ApprovalOperator', 'HITLEntryOperator' in standard provider (#52868)``
```

Comment:
If thisis added it should mention it is only working in future Airflow 3.1

Reply:
We don't specify such info on change logs.
it should be noted in the docs of the feature

Reply:
+1